pkg/aggregator/aggregator.go

```diff
@@ -107,13 +107,19 @@ var (
 	aggregatorHostnameUpdate                   = expvar.Int{}
 	aggregatorOrchestratorMetadata             = expvar.Int{}
 	aggregatorOrchestratorMetadataErrors       = expvar.Int{}
+	aggregatorDogstatsdContexts                = expvar.Int{}
+	aggregatorDogstatsdBucketsContexts         = expvar.Int{}
 
 	tlmFlush = telemetry.NewCounter("aggregator", "flush",
 		[]string{"data_type", "state"}, "Number of metrics/service checks/events flushed")
 	tlmProcessed = telemetry.NewCounter("aggregator", "processed",
 		[]string{"data_type"}, "Amount of metrics/services_checks/events processed by the aggregator")
 	tlmHostnameUpdate = telemetry.NewCounter("aggregator", "hostname_update",
 		nil, "Count of hostname update")
+	tlmContexts = telemetry.NewCounter("aggregator", "contexts",
+		nil, "Count the number of contexts in the aggregator")
```

Comment:
you need to specify the tag key here (the value will be `dogstatsd`), so maybe:

```suggestion
		[]string{"aggregator"}, "Count the number of contexts in the aggregator")
```